docs/snapshotting/snapshot-editor.md

```diff
@@ -0,0 +1,61 @@
+# Snapshot editor
+
+The `snapshot-editor` is a program for modification of Firecracker snapshots.
+
+## Prior knowledge
+
+Firecracker snapshot consists of 2 files:
+
+- vmstate file: file with Firecracker internal data such as vcpu states,
+  devices states etc.
+- memory file: file with guest memory.
+
+`snapshot-editor` allows to modify each of these files separately.
+
+**Note**
+As of right now, `snapshot-editor` has functionality to edit `aarch64` snapshots.
+
+## Usage
+
+### `edit-vmstate` command
+
+#### `remove-regs` subcommand (aarch64 only)
+
+This command is used to remove specified registers from vcpu states inside
+vmstate snapshot file.
+
+Usage:
+
+```bash
+snapshot-editor edit-vmstate remove-regs \
+    --vmstate-path <VMSTATE_PATH> \
+    --output-path <OUTPUT_PATH> \
+    [REGS]...
+```
+
+Example:
+
+```bash
+./snapshot-editor edit-vmstate remove-regs \
+    --vmstate-path ./vmstate_file \
+    --output-path ./new_vmstate_file \
+    0x1 0x2
+```
```

Comment:
Maybe this can briefly explain what the tool does? e.g. reads `vmstate_file`, removes the registers 0x1 and 0x2 and then writes the modified file to `new_vmstate_file`

Reply:
There is a small description of the command. I think it is sufficient.